kolibri/content/api.py

```diff
@@ -143,9 +106,20 @@ def get_missing_files(channel_id=None, content=None, **kwargs):
     """
     if content.kind == 'topic':
         all_end_nodes = leaves(channel_id=channel_id, content=content)
-        return KolibriContent.File.objects.using(channel_id).filter(available=False, format__contentmetadata__in=all_end_nodes)
+        return KolibriContent.File.objects.using(channel_id).filter(available=False, contentmetadata__in=all_end_nodes)
```

Comment:
When/how is this function used? Do we want it to be returning File objects that are for a supplemental preset?